packages/editor/src/worker/worker.ts

```
@@ -1,138 +1,252 @@
-      break;
-    }
-    case "Resample": {
-      const { variation } = data;
-      const resampled = resample({ ...currentState, variation });
-      optimize(insertPending(resampled));
+
+    case WorkerState.Compiled:
+      switch (data.tag) {
+        case "OptimizingReq":
+          log.info("Received OptimizingReq in state Compiled");
+          const stateWithoutLabels: PartialState = unoptState;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)